.github/workflows/build-process.yml

```
@@ -52,3 +57,15 @@ jobs:
         github_token: ${{ secrets.GITHUB_TOKEN }}
         publish_dir: ./docs/_site
         publish_branch: gh-pages
+  check_links:
+    needs: [build_and_publish]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download math result for job 2
```

```suggestion
    - name: Download site artifacts
```

That's what I get for pulling from an example. 😆